metrics/network/README.md

```diff
@@ -20,7 +20,10 @@ maximum number of (smallest sized) packets allowed through the transports.
 
 - `network-metrics-nuttcp.sh` measures the UDP bandwidth using nuttcp. This tool
 shows the speed of the data transfer for the UDP protocol.
- 
+
+- `network-latency.sh` measures the latency using ping. The ping utility measures
+how long it takes one packet to get from one point to another.
+
 ## Running the tests
```

Comment:
@amshinde this tell us how to run the tests, should I put more information?

Reply:
Yes, lets add it for every test in the future.